command/agent/agent_endpoint.go

```diff
@@ -36,6 +36,11 @@ func (s *HTTPServer) AgentSelf(resp http.ResponseWriter, req *http.Request) (int
 	}, nil
 }
 
+func (s *HTTPServer) AgentReload(resp http.ResponseWriter, req *http.Request) (interface{}, error) {
+	_, err := s.agent.command.handleReload(s.agent.config)
```

Comment:
Similar for these as the monitor - let's add the Raft check with a TODO so this needs the operator read privs in master. These are pretty serious otherwise.